### ISSUE TITLE
[mobx-undecorate] Removes not mobx decorators

### ISSUE THREAD

**Issue by pustovalov** [mobxjs/mobx#2558]
Code:

```
@withRouter
@observer
export class Component extends React.PureComponent {
```

**Intended outcome:**

```
export const Component = withRouter(observer(class Component extends React.PureComponent {}))
```

or throw an error

**Actual outcome:**

at the moment silently changes the code to this
```
export const Component = observer(class Component extends React.PureComponent {})
```

**How to reproduce the issue:**

run `npx mobx-undecorate`

**Versions**

mobx-undecorate: 1.0.2

**Comment by FredyC** [comment]
It would be very helpful if you can contribute a PR with a failing test for this. It will be much easier to fix without breaking anything else.

**Comment by mweststrate** [comment]
Released as mobx-undecorate@1.0.3
